x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/formatter.ts

```diff
@@ -15,7 +15,7 @@ export const formatter = (fields: Record<string, any>) => {
   Object.keys(defaults).map((key) => {
     /* split key names on dot to handle dot notation fields,
      * which are changed to nested fields by react-hook-form */
-    monitorFields[key] = get(fields, key.split('.')) || defaults[key as ConfigKey];
+    monitorFields[key] = get(fields, key.split('.')) ?? defaults[key as ConfigKey];
```

Comment:
Couldn't figure out why it was only possible to `enable` my monitors, but not `disable` them 😆

Reply:
Gosh, Javascript strikes again.